services/modrinth/modrinth-version.service.js

```diff
@@ -0,0 +1,28 @@
+import { renderVersionBadge } from '../version.js'
+import { BaseModrinthService, documentation } from './modrinth-base.js'
+
+export default class ModrinthVersion extends BaseModrinthService {
+  static category = 'version'
+
+  static route = {
+    base: 'modrinth/version',
+    pattern: ':projectId',
+  }
+
+  static examples = [
+    {
+      title: 'Modrinth Version',
+      namedParams: { projectId: 'AANobbMI' },
+      staticPreview: renderVersionBadge({ version: '0.4.4' }),
+      documentation,
+    },
+  ]
+
+  static defaultBadgeData = { label: 'version' }
+
+  async handle({ projectId }) {
+    const { 0: latest } = await this.fetchVersions({ projectId })
```

Comment:
Do we definitely know that the first item in the array will be the latest version? The docs are not explicit on this.

Reply:
I don't see a reference to this on the docs, however, I asked them in their Discord server and they confirmed this is the case for the v2 API.
![image](https://user-images.githubusercontent.com/52843537/202295997-a7c458a7-3ce6-4e4f-bfb5-0aff770ccf70.png)